src/lucky_migrator/create_table_statement.cr

```diff
@@ -73,3 +96,43 @@ class LuckyMigrator::CreateTableStatement
     end
   end
 end
+
+# Encapsulates the building of an index string.
+# Provides an add_to method that accepts a reference to an array and adds the
+# index string or raises an exception if already added.
+#
+# ### Usage
+#
+# ```
+# indices = [] of String
```

Comment:
Could you give a real example of strings so it's more clear what the indices array would look like in practice?